source/_patterns/02-organisms/components/highlight~one-item.json

```diff
@@ -0,0 +1,22 @@
+{
+  "heading": {
+    "heading": "Highlights"
+  },
+  "items": [
+    {
+      "title": "[Do not use this base pattern directly: see the following variants of this pattern for usage]",
```

Comment:
You should use an example that looks more real.

Reply:
Real data is added in all examples for one, two and full items